cross_list_courses/templates/cross_list_courses/index.html

```diff
@@ -0,0 +1,59 @@
+{% load collections %}
+{% load django_helpers %}
+{% load static %}
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport"
+          content="width=device-width, initial-scale=1, user-scalable=no">
+    <link href="/static/favicon.ico" rel="icon" type="image/x-icon">
+    <link rel="stylesheet" type="text/css"
+                           href="{% static 'font-awesome-4.2.0/css/font-awesome.min.css' %}">
+    <link rel="stylesheet" type="text/css"
+                           href="{% static 'bootstrap-3.3.4/css/bootstrap.min.css' %}"
+                           media="screen"/>
+    <link rel="stylesheet" type="text/css" href="{% static 'datatables-1.10.7/media/css/jquery.dataTables.min.css' %}" media="screen"/>
+    <link rel="stylesheet" type="text/css" href="{% static 'datatables-bootstrap3/BS3/assets/css/datatables.css' %}" media="screen"/>
+    <link rel="stylesheet" type="text/css" href="{% static 'cross_list_courses/css/index.css' %}">
+
+    {% if debug %}
+    <script src="{% static 'jquery-1.10.2/jquery-1.10.2.js' %}"></script>
+    <script src="{% static 'datatables-1.10.7/media/js/jquery.dataTables.js' %}"></script>
+    <script src="{% static 'underscore-1.8.2/underscore.js' %}"></script>
+    <script src="{% static 'bootstrap-3.3.4/js/bootstrap.js' %}"></script>
+    <script src="{% static 'angular-1.3.9/angular.js' %}"></script>
+    <script src="{% static 'angular-1.3.9/angular-route.js' %}"></script>
+    <script src="{% static 'angular-1.3.9/angular-sanitize.js' %}"></script>
+    <script src="{% static 'angular-ui-bootstrap-0.14.3/ui-bootstrap-tpls-0.14.3.js' %}"></script>
+    <script src="{% static 'djangular/js/django-angular.js' %}"></script>
+    <script src="{% static 'angular-datatables-0.5.1/angular-datatables.js' %}"></script>
+    {% else %}
+    <script src="{% static 'jquery-1.10.2/jquery-1.10.2.min.js' %}"></script>
+    <script src="{% static 'datatables-1.10.7/media/js/jquery.dataTables.min.js' %}"></script>
+    <script src="{% static 'underscore-1.8.2/underscore.min.js' %}"></script>
+    <script src="{% static 'bootstrap-3.3.4/js/bootstrap.min.js' %}"></script>
+    <script src="{% static 'angular-1.3.9/angular.min.js' %}"></script>
+    <script src="{% static 'angular-1.3.9/angular-route.min.js' %}"></script>
+    <script src="{% static 'angular-1.3.9/angular-sanitize.min.js' %}"></script>
+    <script src="{% static 'angular-ui-bootstrap-0.14.3/ui-bootstrap-tpls-0.14.3.min.js' %}"></script>
+    <script src="{% static 'djangular/js/django-angular.min.js' %}"></script>
+    <script src="{% static 'angular-datatables-0.5.1/angular-datatables.min.js' %}"></script>
+    {% endif %}
+
+    <script src="{% static 'datatables-bootstrap3/BS3/assets/js/datatables.js' %}"></script>
```

Comment:
Hmm, I wonder if we still need this? Seems like [this version](https://github.com/DataTables/Plugins/tree/master/integration/bootstrap/3) is newer, and the readme for it explains that bootstrap supported is now integrated into the datatables core library. In fact there's a newer version of datatables where a LOT of stuff has changed it appears: https://github.com/DataTables/DataTables/compare/1.10.7...1.10.11. There's a new version of angular-datatables as well: https://github.com/l-lin/angular-datatables/releases.